src/components/CoinsSelectAutocomplete/CoinsSelectAutocomplete.tsx

```diff
@@ -423,9 +34,9 @@ export function CoinsSelectAutocomplete({
   return (
     <SelectAutocomplete
       Label="Currency"
-      HelperText={<BackToHomeHelperText env={safeBitgoEnv} />}
+      HelperText={helperText}
```

Comment:
In future usage , this component will be rendered twice. It will look very odd to have this helper text appear twice in the same form.